src/leagues/management/commands/setup.py

```diff
@@ -10,31 +10,23 @@
 from base.middleware import env
 from base.models import Value
 from districts.models import District
-from leagues.models import League, LeagueName, Season
+from leagues.models import League, LeagueName, Season, YouthUndecidableError
 from teams.models import Team
 
-LOGGER = logging.getLogger('hbscorez')
-
+logger = logging.getLogger('hbscorez')
```

Comment:
shouldn't lowercase this module level constant, see https://github.com/pylint-dev/pylint/issues/2166
also if this was to be renamed, all the other logger variables in the project should also be renamed